db/write_thread.cc

```diff
@@ -656,12 +657,57 @@ void WriteThread::ExitAsMemTableWriter(Writer* /*self*/,
   SetState(leader, STATE_COMPLETED);
 }
 
+void WriteThread::SetMemWritersEachStride(Writer* w) {
+  WriteGroup* write_group = w->write_group;
+  Writer* last_writer = write_group->last_writer;
+
+  // The stride is the same for each writer in write_group, so w will
+  // call the writers with the same number in write_group mod total size
+  size_t stride = static_cast<size_t>(sqrt(write_group->size));
```

Comment:
internal linter complaint: "Unqualified sqrt call may result in a call to the C library function with the argument implicitly converted to double, leading to worse performance. If you are not calling the C library function consider qualifying the call with the namespace.Otherwise, use std::sqrt instead"

Reply:
std::sqrt is used instead.